src/config.ts

```diff
@@ -2640,6 +2646,13 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 				coinGeckoId: 'pstake-finance',
 				coinImageUrl: window.location.origin + '/public/assets/tokens/pstake.png',
 			},
+			{
```

Comment:
this definition of WTBC here can be omitted here. This whole section is a duplicate that will be removed.

Reply:
Removed!